cns/NetworkContainerContract.go

```diff
@@ -80,8 +80,10 @@ const (
 	InfraNIC NICType = "InfraNIC"
 	// Delegated VM NICs are projected from VM to container network namespace
 	DelegatedVMNIC NICType = "DelegatedVMNIC"
```

Comment:
Should we also change the `InfraNIC` & `DelegatedVMNIC` into `NodeNetworkInterfaceInfraNIC` & `NodeNetworkInterfaceDelegatedVMNIC`? @kmurudi

Reply:
Confirmed with Quang and renamed `DelegatedVMNIC`.